types.go

```diff
@@ -90,7 +95,7 @@ type JSONWebKey struct {
 }
 
 type clientData struct {
-	Type               string      `json:"typ"`
```

Comment:
Pretty sure this isn't a typo. If you check the [raw message formats](https://fidoalliance.org/specs/u2f-specs-1.0-bt-nfc-id-amendment/fido-u2f-raw-message-formats.html) spec (search `cid_pubkey` to find the right section), its specified as 
> DOMString typ

You can also see in the examples
```json
{"typ":"navigator.id.finishEnrollment","challenge":"vqrS6WXDe1JUs5_c3i4-LkKIHRr-3XVb3azuA5TifHo","cid_pubkey":{"kty":"EC","crv":"P-256","x":"HzQwlfXX7Q4S5MtCCnZUNBw3RMzPO9tOyWjBqRl4tJ8","y":"XVguGFLIZx1fXg3wNqfdbn75hi4-_7-BxhMljw42Ht4"},"origin":"http://example.com"}
```

Reply:
Ah, OK. Addressed in latest commit.